src/server.c

```diff
@@ -3080,6 +3083,26 @@ void propagatePendingCommands() {
     redisOpArrayFree(&server.also_propagate);
 }
 
+/* Reset the error stats count and set cmd stats in case there are any errors
+ * that happened sense the last snapshot. the flags indicates what stats need
+ * to be updated, options are:
+ * * ERROR_COMMAND_REJECTED - update rejected_calls
+ * * ERROR_COMMAND_FAILED - update failed_calls
+ *
+ * If cmd is NULL, not stats are updated. */
+void resetErrorCountSnapshot(struct redisCommand *cmd, int flags) {
```

Comment:
i think the "reset" is the side effect of this function.
i.e. internally, it uses a static variable to avoid counting the same error twice.
but it's name should probably be `incrCommandFailedCalls()`.
then mention that `cmd` can be NULL to indicate an error was counted elsewhere.

Reply:
Fixed